Alignment/CommonAlignmentProducer/python/ALCARECOTkAlMuonIsolatedHI_cff.py

```diff
@@ -1,27 +1,21 @@
-# AlCaReco for track based alignment using isolated muon tracks
+# AlCaReco for track based alignment using isolated muon tracks - relaxed cuts for PbPb collisions
 import FWCore.ParameterSet.Config as cms
 
 from Alignment.CommonAlignmentProducer.ALCARECOTkAlMuonIsolated_cff import *
 
 ALCARECOTkAlMuonIsolatedHIHLT = ALCARECOTkAlMuonIsolatedHLT.clone(
-    eventSetupPathsKey = 'TkAlMuonIsolated'
+    eventSetupPathsKey = 'TkAlMuonIsolatedHI'
     )
```

Comment:
We are, I checked by doing:

runTheMatrix.py -l 140.51

The outcome of which was:

140.51_RunHI2010+RunHI2010+REPACKHID+RECOHID11St3+HARVESTDHI Step0-PASSED Step1-PASSED Step2-PASSED Step3-PASSED  - time date Thu Oct  6 17:49:41 2016-date Thu Oct  6 17:45:09 2016; exit: 0 0 0 0
1 1 1 1 tests passed, 0 0 0 0 failed